dockerfiles/che/entrypoint.sh

```diff
@@ -229,11 +229,11 @@ init() {
 
   if [ -z "$CHE_HOME" ]; then
     if [ -f "/assembly/tomcat/bin/catalina.sh" ]; then
-      echo "Found custom assembly in /assembly"
       export CHE_HOME="/assembly"
+      echo "Found custom assembly in ${CHE_HOME}"
     else
-      echo "Using embedded assembly."
       export CHE_HOME=$(echo /home/user/eclipse-che/)
+      echo "Using embedded assembly in ${CHE_HOME}."
```

Comment:
Do we nee trailing slash at the end? I think it's not needed.
wdyt?

Reply:
I mean `echo /home/user/eclipse-che/` ->`echo /home/user/eclipse-che`

Reply:
No idea, it's YOUR entrypoint.sh script. I just copy it to downstream :D